src/rdkafka_cgrp.c

```diff
@@ -1066,10 +1066,7 @@ rd_kafka_cgrp_partitions_fetch_start0 (rd_kafka_cgrp_t *rkcg,
                                  * no forwarding queue already set. */
                                 rd_kafka_toppar_op_fetch_start(
                                         rktp, rktpar->offset,
-                                        ~(rktp->rktp_fetchq->rkq_flags &
-                                                RD_KAFKA_Q_F_FWD_APP) ?
-                                                        rkcg->rkcg_q :
```

Comment:
I think you should still pass rkcg_q here and just avoid the q_fwd_set() in toppar_fetch_start() if _Q_FWD_APP is set.